gcp_pilot/calendar.py

```diff
@@ -188,6 +189,7 @@ def _build_time_field(dt):
             "description": description,
             "start": _build_time_field(dt=start_at),
             "end": _build_time_field(dt=end_at),
+            "sendNotifications": send_notifications,
```

Comment:
It seems to be deprecated, they recommend `sendUpdates`, which is a string.
In order to support that, we should add the choices as a Enum to crate an easier interface.